### PR TITLE
All Core Devs Meeting 113

### PR DESCRIPTION
**Pull request by KenMan79**
Transcribe of the All Core Devs Meeting 113 held on May 14th, 2021 at 14:00 UTC by Kenneth Luster

### CODE REVIEW

File: All Core Devs Meetings/Meeting Template.md [hunk 1]
```diff
@@ -1,14 +1,593 @@
-# All Core Devs Meeting #
-### Meeting Date/Time: Friday xx/xx/17 at 14:00 UTC
-### Meeting Duration 2 hours
-### [Audio/Video of the meeting]()
+# All Core Devs Meeting 113 #309
+### Meeting Date/Time: Friday 05/14/2021 at 14:00 UTC
+### Meeting Duration: 1:20:17
+### [Audio/Video of the meeting](https://youtu.be/H_T2nNrTuWQ![image](https://user-images.githubusercontent.com/51566205/118755269-2ee99b00-b82e-11eb-80ba-ec5db201852d.png)
+)
 ### [Reddit thread]()
```
Comment (timbeiko) [inline All Core Devs Meetings/Meeting Template.md:6]: What is this for ?

**Comment by timbeiko** [comment]
@KenMan79 small comment, but otherwise looks good.

File: All Core Devs Meetings/Meeting Template.md (hunk 1 above)
Comment (KenMan79) [reply All Core Devs Meetings/Meeting Template.md:6]: This is for the Transcription of Friday, May 14th  All Core Devs #113 Meeting. 
(If I made a PR in the incorrect location please let me know)

Comment (timbeiko) [reply All Core Devs Meetings/Meeting Template.md:6]: I meant what is the Reddit link for ? I didn't know we typically added those, and right now the link is empty.

Comment (poojaranjan) [reply All Core Devs Meetings/Meeting Template.md:6]: @timbeiko Please hold on merging to this file.

Comment (KenMan79) [reply All Core Devs Meetings/Meeting Template.md:6]: Yeah, did not place in Reddit. I left the place intact for if someone has Reddit and able to fill the spot. As well as for someone who refers to this as an example.

**Comment by poojaranjan** [comment]
@KenMan79 
It seems you're suggesting changes to the template which is supposed to be used as a reference. 
To submit the PR for ACD meeting notes, please create a new file using the template and create a pull request for [ACD PM  folder](https://github.com/ethereum/pm/tree/master/All%20Core%20Devs%20Meetings).

File: All Core Devs Meetings/Meeting Template.md (hunk 1 above)
Comment (poojaranjan) [reply All Core Devs Meetings/Meeting Template.md:6]: This template can be ignored as may not have been updated in a while. Please use the other [template](https://github.com/ethereum-cat-herders/PM/blob/master/Meeting_Notes_Template.md#meeting-name-ethereum-core-devs-meeting--ethereum-20-implementers-call-ethereum-cat-herders-meeting-notes)  shared with you earlier.

**Pull request closed without merging**